tests/ops/api/v1/endpoints/test_dataset_endpoints.py

```diff
@@ -192,7 +192,7 @@ def test_put_validate_dataset_invalid_length(
         assert response.status_code == 422
         assert (
             json.loads(response.text)["detail"][0]["msg"]
-            == "Illegal length (-1). Only positive non-zero values are allowed."
+            == "ensure this value is greater than 0"
```

Comment:
Validation of field length was moved to Fideslang. It was updated to use a built-in Pydantic validator instead of our own custom validation to enforce positive non-zero models, so the error message has changed.